lib/solarwinds_apm/otel_lambda_config.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# © 2023 SolarWinds Worldwide, LLC. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at:http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+require 'solarwinds_apm/constants'
+require 'solarwinds_apm/api'
+require 'solarwinds_apm/support'
+require 'solarwinds_apm/opentelemetry'
+
+module SolarWindsAPM
+  # OTelLambdaConfig module
+  module OTelLambdaConfig
+    def self.initialize
+      return unless defined?(::OpenTelemetry::SDK::Configurator)
+
+      ENV['OTEL_TRACES_EXPORTER'] = 'none' if ENV['OTEL_TRACES_EXPORTER'].to_s.empty?
+
+      ::OpenTelemetry::SDK.configure do |c|
+        c.resource = { 'sw.apm.version' => SolarWindsAPM::Version::STRING,
+                       'sw.data.module' => 'apm',
+                       'service.name' => ENV['OTEL_SERVICE_NAME'] || ENV.fetch('AWS_LAMBDA_FUNCTION_NAME', nil) }
+        c.use 'OpenTelemetry::Instrumentation::AwsLambda'
```

Comment:
Since this is `c.use` instead of `c.use_all` does that mean only the lambda instrumentation will be loaded?

Reply:
I thought we'd want to install and enable all instrumentations when instrumenting a lambda function. Meaning if the function made AWS SDK calls or RPC calls, the respective instrumentation would produce spans and give a richer view into the execution.